sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.13.0-beta.1 (Unreleased)
+## 1.14.0 (2021-04-07)
```

Comment:
You're skipping 1.13.0 and going right to 1.14 - was that intentional?

Reply:
```suggestion
## 1.13.0 (2021-04-07)
```